tests/test_sdba/test_adjustment.py

```diff
@@ -388,9 +388,6 @@ def test_cannon_and_diagnostics(self, cannon_2015_dist, cannon_2015_rvs):
 
         assert isinstance(scends, xr.Dataset)
 
-        sim_q_exp = sim.rank(dim="time", pct=True)
-        np.testing.assert_array_equal(sim_q_exp, scends.sim_q)
-
```

Comment:
Note to the reviewer: this test has been removed as it's not doing anything, the computation for `sim_q_exp` would be the exact same thing as the one for `scends.sim_q`, this is only testing if we correctly copypasted the code from the algorithm to the test.